### PR TITLE
Bk/add swift linting

### PR DESCRIPTION
**Pull request by bryankeller**
## Details

Add Swift format support and enforce Airbnb's style guide

## Related Issue

N/A

## Motivation and Context

Automatic linting and style guide enforcement

## How Has This Been Tested

Example app compiles, tests compile and pass

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project. 😎
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by calda — DISMISSED** [review]
Awesome! Glad to have more adoption of our swift package plugin

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added the ability to change the aspect ratio of individual day-of-the-week items
 - Added support for self-sizing month headers
 - Added a new `setContent(_:animated:)` function, enabling developers to perform animated content updates
+- Added Swift format integration to enforce consistent code style
```
Comment (calda) [inline CHANGELOG.md:16]: Should the changelog mention that the package now requires Swift 5.8+?

File: Package.swift [hunk 2]
```diff
@@ -1,25 +1,25 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.8
```
Comment (calda) [inline Package.swift:1]: Should this be kept in sync with the value in the podspec?

File: CHANGELOG.md (hunk 1 above)
Comment (bryankeller) [reply CHANGELOG.md:16]: Oh yeah, good call

**Review by calda — DISMISSED** [review]

**Review by calda — APPROVED** [review]

**Pull request merged**